content/quickstart/Node.js/metrics.md

```diff
@@ -0,0 +1,571 @@
+
+In this quickstart, we’ll gleam insights from code segments and learn how to:
+
+1. Collect metrics using [OpenCensus Metrics](/core-concepts/metrics) and [Tags](/core-concepts/tags)
+2. Register and enable an exporter for a [backend](/core-concepts/exporters/#supported-backends) of our choice
+3. View the metrics on the backend of our choice
+
+## Requirements
+- [Node.js](https://nodejs.org/) 6 or above and `npm` (already comes with Node.js)
+- Google Cloud Platform account and project
+- Google Stackdriver Monitoring enabled on your project
+
+{{% notice tip %}}
+For assistance setting up Stackdriver, [Click here](/codelabs/stackdriver) for a guided codelab.
+{{% /notice %}}
+
+## Installation
+
+OpenCensus: `npm install @opencensus/nodejs`
+
+Stackdriver exporter: `npm install @opencensus/exporter-stackdriver`
+
+## Brief Overview
+By the end of this tutorial, we will do these four things to obtain metrics using OpenCensus:
+
+1. Create quantifiable `metrics` (numerical) that we will **record**
+2. Create [tags](/core-concepts/tags) that we will associate with our metrics
+3. Organize our metrics, similar to writing a report, in to a `View`
+4. Export our views to a backend (Stackdriver in this case)
+
+## Getting Started
+
+{{% notice note %}}
+Unsure how to write and execute Node.js code? [Click here](https://nodejs.org/en/docs/guides/getting-started-guide/).
+{{% /notice %}}
+
+We will be a simple "read-evaluate-print-loop" (REPL) app. In there we'll collect some metrics to observe the work that is going on within this code, such as:
+
+- Latency per processing loop
+- Number of lines read
+- Line lengths
+
+First, create a file called `repl.js`. Next, put the following code inside of `repl.js`:
+
+{{<highlight javascript>}}
+import * as fs from "fs";
+import * as readline from "readline";
+
+// Creates a stream to read our file
+const stream = fs.createReadStream("./test.txt");
+
+// Creates an interface to read and process our file line by line
+const lineReader = readline.createInterface({ input: stream });
+
+// REPL is the read, evaluate, print and loop
+lineReader.on("line", function (line) {       // Read
+  const processedLine = processLine(line);    // Evaluate
+  console.log(processedLine);                // Print
+});
+
+/**
+ * Takes a line and process it.
+ * @param {string} line The line to process
+ */
+function processLine(line) {
+  // Currently, it just capitalizes it.
+  return line.toUpperCase();
+}
+{{</highlight>}}
+
+You can run the code via `node repl.js`.
+
+## Enable Metrics
+
+### Import Packages
+
+To enable metrics, we'll import a few items from OpenCensus Core package.
+
+{{<tabs Snippet All>}}
+{{<highlight javascript>}}
+import { Stats, MeasureUnit, AggregationType } from "@opencensus/opencensus-core";
+{{</highlight>}}
+
+{{<highlight javascript>}}
+import { Stats, MeasureUnit, AggregationType } from "@opencensus/opencensus-core";
+
+import * as fs from "fs";
```

Comment:
why not just
```javascript
import fs;
```
and ditto for the rest?

Reply:
Using `import "fs" runs the module's global code, but doesn't actually import any values. Also, although here is used the same name, using `import * as name from 'module'` allows you to have more flexibility when using the module.